app/src/main/java/gov/hhs/cdc/trustedintermediary/external/in_memory/KeyCache.java

```diff
@@ -0,0 +1,33 @@
+package gov.hhs.cdc.trustedintermediary.external.in_memory;
+
+import gov.hhs.cdc.trustedintermediary.wrappers.Cache;
+import java.util.Map;
+import java.util.concurrent.ConcurrentHashMap;
+
+/** Stores keys from Azure to optimize performance. */
```

Comment:
Consider rewriting this JavaDocs to be more generic.

Reply:
Agree.